frontend/src/vars.scss

```diff
@@ -129,7 +129,7 @@ $top_nav_height: 50px;
     max-width: calc(100% - 8px);
 }
 
-// Z-indexes
+// Z-indexes, sync with vars.ts
```

Comment:
The need to make the same change in two places feels like a potential source of hard to diagnose bugs. I don't know what the solution is, but feels like this needs follow on work to resolve